flocker/node/agents/functional/test_gce.py

```diff
@@ -175,3 +191,31 @@ def test_attach_elsewhere_attached_volume(self):
             blockdevice_id=attached_volume.blockdevice_id,
             attach_to=api.compute_instance_id(),
         )
+
+    def test_list_volumes_walks_pages(self):
+        api = gceblockdeviceapi_for_test(self)
+        api._page_size = 1
```

Comment:
:question: It might be worth using `self.patch(api, '_page_size', 1) which will raise an exception if the `_page_size` is attribute is ever removed or renamed.